processor/resourcedetectionprocessor/README.md

```diff
@@ -55,11 +55,19 @@ to read resource information from the [GCE metadata server](https://cloud.google
     * aws.log.group.arns (V4 only)
     * aws.log.stream.names (V4 only)
     * aws.log.stream.arns (V4 only)
+    
+* Amazon Elastic Beanstalk: Reads the AWS X-Ray configuration file available on all Beanstalk instances with [X-Ray Enabled](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environment-configuration-debugging.html).
 
+    * cloud.provider (aws)
+    * cloud.infrastructure_service (ElasticBeanstalk)
+    * deployment.environment
+    * service.instance.id
+    * service.version
+    
 ## Configuration
 
 ```yaml
-# a list of resource detectors to run, valid options are: "env", "system",  "gce", "ec2", "ecs"
+# a list of resource detectors to run, valid options are: "env", "system",  "gce", "ec2", "ecs", "elasticBeanstalk"
```

Comment:
nit: for component identifiers (e.g. processor/k8s_tagger) we do underscores instead of camel case so maybe better to do `elastic_beanstalk`?

Reply:
Ok, I will rename